test/dynamo/test_higher_order_ops.py

```diff
@@ -2916,23 +2920,27 @@ def fn(x):
             """\
 class GraphModule(torch.nn.Module):
     def forward(self, L_x_ : torch.Tensor):
-        child = L_x_
+        arg = L_x_
 
-        _check_randomness_arg = torch._functorch.vmap._check_randomness_arg('error')
+        lazy_load_decompositions = torch._functorch.vmap.lazy_load_decompositions()
 
-        select = child.select(0, 0)
-        vmap_body_0 = self.vmap_body_0
-        vmap_proxy = torch.func.vmap(vmap_body_0, (0,), 0, 'error');  vmap_body_0 = None
-        call = vmap_proxy.__call__(child);  vmap_proxy = child = None
-        return (call,)
+        _saved_tensors_hooks_disable = torch._C._autograd._saved_tensors_hooks_disable("torch.func transforms don't yet support saved tensor hooks. Please open an issue with your use case.")
 
-    class GraphModule(torch.nn.Module):
-        def forward(self, select):
-            sum_1 = select.sum(0)
-            sum_2 = select.sum(1);  select = None
-            add = sum_1 + sum_2;  sum_1 = sum_2 = None
-            add_1 = add + 3;  add = None
-            return add_1
+        _vmap_increment_nesting = torch._C._functorch._vmap_increment_nesting(3, 'error')
+
+        _add_batch_dim = torch._C._functorch._add_batch_dim(arg, 0, 1);  arg = None
+
```

Comment:
Another idea for test cases: we should remove all of the skipIfTorchDynamo/xfailIfTorchDynamo from https://github.com/pytorch/pytorch/blob/main/test/functorch/test_vmap.py and fix the failures. It doesn't have to happen in this PR (as long as this PR doesn't add more of them).

Reply:
Working on this one at the moment.

Reply:
This is blocked until https://github.com/pytorch/pytorch/issues/116264 gets fixed